awscrt/io.py

```diff
@@ -276,7 +290,65 @@ def create_client_with_mtls(cert_buffer, key_buffer):
         opt.certificate_buffer = cert_buffer
         opt.private_key_buffer = key_buffer
 
-        opt.verify_peer = True
+        return opt
+
+    @staticmethod
+    def create_client_with_mtls_pkcs11(*,
+                                       pkcs11_lib: 'Pkcs11Lib',
+                                       user_pin: str,
+                                       slot_id: int = None,
+                                       token_label: str = None,
+                                       private_key_label: str = None,
+                                       cert_file_path: str = None,
+                                       cert_file_contents=None):
```

Comment:
cert_file_contents=None -> cert_file_contents = None
add spaces for uniformity with other arguments?

Reply:
the autoformatter made it look like this.

The reason they're inconsistent is that every argument **except** `cert_file_contents` has a [type hint](https://docs.python.org/3/library/typing.html). So the autoformatter is like "this is how I format default values with type hints, and this is how I format default values without type hints". It's too much effort to fight the autoformatter.

I didn't put a type hint on `cert_file_contents` because it just needs to be a "bytes-like object" and there's no appropriate type hint for that right now: see https://bugs.python.org/issue27501